django/library/serializers.py

```diff
@@ -37,7 +37,8 @@ class CodebaseReleaseSerializer(serializers.ModelSerializer):
     programming_languages = home_serializers.TagSerializer(many=True)
 
     def get_relative_url(self, obj) -> str:
-        return reverse_lazy('library:codebaserelease-detail', kwargs={'release_pk': obj.codebase_id, 'pk': obj.id})
+        return reverse_lazy('library:codebaserelease-detail',
```

Comment:
Would recommend delegating to `CodebaseRelease.get_absolute_url` instead and define it there centrally. See https://docs.djangoproject.com/en/1.11/ref/models/instances/#get-absolute-url for more details